common/persistence/visibility/store/query/interceptors_test.go

```diff
@@ -50,7 +50,7 @@ func (t *testNameInterceptor) Name(name string, usage FieldNameUsage) (string, e
 	return name + "1", nil
 }
 
-func (t *testValuesInterceptor) Values(name string, values ...interface{}) ([]interface{}, error) {
+func (t *testValuesInterceptor) Values(name string, fieldName string, values ...interface{}) ([]interface{}, error) {
```

Comment:
Please, add tests in the interceptor to check the conversion is being done correctly for both name and values.

Reply:
I already have the following tests:

```
// Verifies the nameInterceptor correctly transforms ScheduleID to WorkflowID and stores the transformation.
func (s *QueryInterceptorSuite) TestNameInterceptor_ScheduleIDToWorkflowID() {
	ni := s.createMockNameInterceptor(nil)

	fieldName, err := ni.Name(searchattribute.ScheduleID, query.FieldNameFilter)
	s.NoError(err)
	s.Equal(searchattribute.WorkflowID, fieldName)
}

// Ensures the valuesInterceptor applies the ScheduleID to WorkflowID transformation,
// including prepending the WorkflowIDPrefix.
func (s *QueryInterceptorSuite) TestValuesInterceptor_ScheduleIDToWorkflowID() {
	vi := NewValuesInterceptor(
		"test-namespace",
		searchattribute.TestNameTypeMap,
	)

	values, err := vi.Values(searchattribute.ScheduleID, searchattribute.WorkflowID, "test-schedule-id")
	s.NoError(err)
	s.Len(values, 1)
	s.Equal(primitives.ScheduleWorkflowIDPrefix+"test-schedule-id", values[0])
}
```

Is there another test you had in mind?

Reply:
Ah, ok. I missed that.